cypress/e2e/signup_signin_logout.spec.js

```diff
@@ -48,12 +51,12 @@ describe('UI test for sign-up, login and logout', () => {
 
     // sign-up
     it('should allow a visitor to sign-up', () => {
-        cy.ui_sign_up()
+        cy.ui_sign_up(userName, password)
```

Comment:
In this spec following tests don't work:

1. should allow a visitor to logout
2. should show login errors for login with invalid credentials